### ISSUE TITLE
larger icons?

### ISSUE THREAD

**Issue by mattiLeBlanc** [planet-training/material-design-icons#5]
Hi,

As I understand it icon-sets that are pulled in are 24px versions.
Printing them in your markup goes like this:
<md-icon md-svg-icon="navigation:ic_menu_24px" aria-label="Menu"></md-icon>

Looking at the Angular Material Docs ( https://material.angularjs.org/latest/#/api/material.components.icon/directive/mdIcon)
you set the icon size when you declare the icons sets and you don't need to reference the icon as ic_[name]_24px.

This was a bit confusing for me when I use your package.
Maybe you can add in the docs an example how to call an icon.

Also, if you would want to have the 48px version of the icon, you would have to include the material icons your self as documenten in the angular material md-icon page, right?

**Comment by donedgardo** [comment]
Yes, I can confirm trying to get bigger icons with "ic_[name]_48px" won't work.

**Comment by mattiLeBlanc** [comment]
when I look at my network tab, I see the icons sets that are loaded in the sprite are all set to 24px only. So set is fixed. You probably have to duplicate the set and make your own with a bigger size

**Comment by mxab** [comment]
we currently only repackage the sprite files that come with the bower package. I haven't compared the 24 and 48 files, do they contain more detail?

**Comment by dohomi** [comment]
I'd recommend to use the package: https://github.com/planet-training/material-design-icons-font which is font based version of the iconset and recommended by the Google icon team
